langfuse/callback/langchain.py

```diff
@@ -1139,6 +1139,11 @@ def _parse_usage(response: LLMResult):
                     break
```

Comment:
I would suggest adding
```suggestion
                    if llm_usage:
                        break
```

or even better modify the previous condition to:

```
                if generation_chunk.generation_info and (
                        generation_chunk.generation_info.get("usage_metadata", None)
                    ):
```

as in my testing with ChatVertexAI `generation_info` contains empty `usage_metadata` while `message` contains valid one.